dev/helm/templates/deployment.yaml

```
@@ -0,0 +1,77 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "wiki.fullname" . }}
+  labels:
+    {{- include "wiki.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "wiki.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "wiki.selectorLabels" . | nindent 8 }}
+    spec:
+    {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      serviceAccountName: {{ include "wiki.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+            {{ if .Values.postgresql.enabled }}
+            - name: DB_TYPE
+              value: postgres
+            {{ end }}
+            - name: DB_HOST
+              value: {{ template "wiki.postgresql.host" . }}
+            - name: DB_PORT
+              value: "{{ default "5432" .Values.postgresql.postgresqlPort }}"
+            - name: DB_NAME
+              value: {{ default "wiki" .Values.postgresql.postgresqlDatabase }}
+            - name: DB_USER
+              value: {{ default "wiki" .Values.postgresql.postgresqlUser }}
+            - name: DB_PASS
+              valueFrom:
+                secretKeyRef:
+                {{- if .Values.postgresql.existingSecret }}
+                  name: {{ .Values.postgresql.existingSecret }}
+                {{- else }}
+                  name: {{ template "wiki.postgresql.secret" . }}
+                {{- end }}
+                  key: {{ template "wiki.postgresql.secretKey" . }}
+          ports:
+            - name: http
+              containerPort: 3000
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /
+              port: http
+          readinessProbe:
+            httpGet:
+              path: /
+              port: http
```

The `/healthz` endpoint would be better suited.

```suggestion
          livenessProbe:
            httpGet:
              path: /healthz
              port: http
          readinessProbe:
            httpGet:
              path: /healthz
              port: http
```